docs/docs/guides/platform/weave-self-managed.md

```diff
@@ -21,282 +21,383 @@ Although the deployment process sets up a fully functional ClickHouseDB instance
+          storageClassName: standard-rwo
+
+  configuration:
+    # Refer to: https://clickhouse.com/docs/operations/server-configuration-parameters/settings#zookeeper
+    zookeeper:
+      nodes:
+        - host: chk-wandb-keeper-0-0.clickhouse.svc.cluster.local
+          port: 2181
+        - host: chk-wandb-keeper-0-1.clickhouse.svc.cluster.local
+          port: 2181
+        - host: chk-wandb-keeper-0-2.clickhouse.svc.cluster.local
+          port: 2181
+      # Other configurations
+      # session_timeout_ms: 30000
+      # operation_timeout_ms: 10000
+
+    # Users configuration: https://clickhouse.com/docs/operations/configuration-files#user-settings
+    # Tip for password :
+    # sha256sum <<< weave123 OR echo -n weave123 | sha256sum OR printf "weave123" | sha256sum
+    # It wil turn into <password_sha256_hex>...</password_sha256_hex> in user config
+    users:
+      weave/password_sha256_hex: e419535466c5c9c280a157975503187bf27f94093a35a9847ed5249d3df54b4b
+      #weave/password: <PASSWORD> #for cleartext password
+      weave/password:   # to use k8s secrets for "weave" user password
+        valueFrom:
+          secretKeyRef:
+            name: <db-password>
+            key: <password>
+      weave/access_management: 1
+      weave/profile: default
+      weave/networks/ip:
+        - 0.0.0.0/0
+
+    # Settings configuration: https://clickhouse.com/docs/operations/server-configuration-parameters/settings
+    # This can be here or in a config file
+    settings:
+      {}
+      # prometheus/metrics: 1
+      # prometheus/endpoint: /metrics
+      # prometheus/port: 9191
+      # prometheus/events: 1
+      # prometheus/asynchronous_metrics: 1
+
+    clusters:
+      - name: clickhouse
+        layout:
+          shardsCount: 1
+          replicasCount: 3
+        templates:
+          podTemplate: clickhouse
+          dataVolumeClaimTemplate: data
+
+    files:
+      config.d/network_configuration.xml: |
+        <clickhouse>
+            <network_configuration>
+                <listen_host>0.0.0.0</listen_host>
+                <listen_host>::</listen_host>
+            </network_configuration>
+        </clickhouse>
+      config.d/logger.xml: |
+        <clickhouse>
+            <logger>
+                <level>information</level>
+            </logger>
+        </clickhouse>
+      config.d/remote_servers.xml: |
+        <clickhouse>
+          <remote_servers>
+            <weave_cluster>
+              <shard>
+                <replica>
+                  <host>chi-wandb-clickhouse-0-0.clickhouse.svc.cluster.local</host>
+                  <port>9000</port>
+                </replica>
+                <replica>
+                  <host>chi-wandb-clickhouse-0-1.clickhouse.svc.cluster.local</host>
+                  <port>9000</port>
+                </replica>
+                <replica>
+                  <host>chi-wandb-clickhouse-0-2.clickhouse.svc.cluster.local</host>
+                  <port>9000</port>
```

Comment:
This also depends on the name assigned to the deployment; maybe it would be good to generalize or explain why these names are like this.